frame/system/src/lib.rs

```diff
@@ -337,10 +341,8 @@ impl From<sp_version::RuntimeVersion> for LastRuntimeUpgradeInfo {
 decl_storage! {
 	trait Store for Module<T: Trait> as System {
 		/// The full account information for a particular account ID.
-		// TODO: should be hasher(twox64_concat) - will need staged migration
-		// https://github.com/paritytech/substrate/issues/4917
 		pub Account get(fn account):
-			map hasher(blake2_256) T::AccountId => AccountInfo<T::Index, T::AccountData>;
+			map hasher(twox_64_concat) T::AccountId => AccountInfo<T::Index, T::AccountData>;
```

Comment:
```suggestion
			map hasher(blake2_128_concat) T::AccountId => AccountInfo<T::Index, T::AccountData>;
```

This map can get populated with an arbitrary balance transfer, or any other arbitrary implementation of `AccountData`